homework1/tests/task02.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from homework1.task02 import check_fibonacci
+
+
+@pytest.mark.parametrize(
+    ["value", "expected_result"],
+    [([1, 1, 2, 3, 5, 8], True), ([1, 1, 2, 3, 5, 7], False)],
```

Comment:
You should add more corner cases for this test: 0 elements in the list, 1 element, etc.

Reply:
fix